core/build.gradle

```diff
@@ -24,8 +24,15 @@ import org.elasticsearch.gradle.test.RestSpecHack
 apply plugin: 'elasticsearch.build'
 apply plugin: 'com.bmuschko.nexus'
 apply plugin: 'nebula.optional-base'
+apply plugin: "application"
 
 archivesBaseName = 'elasticsearch'
+def runEsHome="${project.buildDir}/run-elastic"
+mainClassName = "org.elasticsearch.bootstrap.Elasticsearch"
+applicationDefaultJvmArgs = ["-Xms256m", "-Xmx1g", "-Djava.awt.headless=true", "-Delasticsearch", "-Des.foreground=yes",
+                             "-ea", "-Des.path.home=" + runEsHome, "-Des.security.manager.enabled=false",
```

Comment:
I've removed the test argument, but the security manager has to stay disabled unfortunately. 

Another way would be take the fresh distribution, unpackage, run. But that sounds like an overkill at this stage.

At least the run scripts will start a node, it can be improved on later.